waiter/src/waiter/util/utils.clj

```diff
@@ -531,3 +544,21 @@
   (if (instance? ExceptionInfo e)
     (ex-info (.getMessage e) (update-fn (ex-data e)) (or (.getCause e) e))
     (ex-info (.getMessage e) (update-fn {}) e)))
+
+(defmacro keys-map
```

Comment:
Nitpick: not sure how I feel about this, tying the variable name to the key of the map. It would be good to know how others in the team feel about this function and `strs-map`.